services/audit-service/src/main/java/datawave/microservice/audit/AuditController.java

```diff
@@ -122,7 +122,7 @@ private boolean sendMessage(AuditParameters parameters) {
             }
             
             boolean success = messageChannel.send(MessageBuilder.withPayload(AuditMessage.fromParams(parameters)).setCorrelationId(auditId).build());
-            
+
```

Comment:
These whitespace changes seem to indicate that the formatter hasn't been run on this file. Can you ensure that you've run the maven build and then include any files that got reformated as a result?

Reply:
I ran the build and no files were reformatted. My IDE is set to remove indents with empty lines (this is after importing the datawave code-style settings). Should empty-line indents be preserved?

Reply:
I think the formatter puts the indents on empty lines. You've saved the file in the IDE after building, then you would get the problem. If you have your IDE closed, build, and see no modified files, then everything's ok.

Reply:
Yeah, I built the target of this PR, and I do end up with a modified AuditController.java file.

Reply:
Confirmed, the issue was an IDE setting. I restored the whitespace and updated my IDE to preserve indents.